### PR TITLE
[DOC]Doris BE development and debugging environment construction

### PR DESCRIPTION
**Pull request by hf200012**
Doris BE development and debugging environment construction

Add installation under ubuntu, dependent installation

Compile on ubuntu 20.04 physical machine, the actual test needs to install these dependencies: autoconf automake libtool autopoint

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — COMMENTED** [review]
Thank for your share, please update the sidebar

File: docs/.vuepress/sidebar/en.js [hunk 1]
```diff
@@ -541,6 +541,7 @@ module.exports = [
         "debug-tool",
         "fe-eclipse-dev",
         "fe-idea-dev",
+		"be-vscode-dev",
```
Comment (morningman) [inline docs/.vuepress/sidebar/en.js:544]: indent, using space instead of tab

File: docs/.vuepress/sidebar/zh-CN.js [hunk 2]
```diff
@@ -546,6 +546,7 @@ module.exports = [
         "debug-tool",
         "fe-eclipse-dev",
         "fe-idea-dev",
+		"be-vscode-dev",
```
Comment (morningman) [inline docs/.vuepress/sidebar/zh-CN.js:549]:
```suggestion
        "be-vscode-dev",
```

Comment (hf200012) [reply docs/.vuepress/sidebar/zh-CN.js:549]: OK

File: docs/zh-CN/developer-guide/be-vscode-dev.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+1. 下载源码
+
+```
+git clone https://github.com/apache/incubator-doris.git 
+```
+
+2. 编译第三方依赖包
+
+```
+ cd /home/zhangfeng/incubator-doris/thirdparty
```
Comment (EmmyMiao87) [inline docs/zh-CN/developer-guide/be-vscode-dev.md:69]:
```suggestion
 cd /home/incubator-doris/thirdparty
```

File: docs/zh-CN/developer-guide/be-vscode-dev.md [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+ ./build-thirdparty.sh
+```
+
+3. 编译doris产品代码
+
+```
+cd /home/zhangfeng/incubator-doris
```
Comment (EmmyMiao87) [inline docs/zh-CN/developer-guide/be-vscode-dev.md:76]:
```suggestion
cd /home/incubator-doris
```

File: docs/zh-CN/developer-guide/be-vscode-dev.md [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+./build.sh  --be #只编译be
+./build.sh  --fe #只编译fe
+./build.sh  --fe --be#同时编译be fe
+./build.sh  --fe --be --clean#删除并同时编译be fe
+./build.sh  --fe  --clean#删除并编译fe
+./build.sh  --be  --clean#删除并编译be
+./build.sh  --be --fe  --clean#删除并同时编译be fe
+```
+
+如果不出意外，应该会编译成功，最终的部署文件将产出到  /home/zhangfeng/incubator-doris/output/ 目录下。如果还遇到其他问题，可以参照 doris 的安装文档 http://doris.apache.org。
```
Comment (EmmyMiao87) [inline docs/zh-CN/developer-guide/be-vscode-dev.md:93]:
```suggestion
如果不出意外，应该会编译成功，最终的部署文件将产出到  /home/incubator-doris/output/ 目录下。如果还遇到其他问题，可以参照 doris 的安装文档 http://doris.apache.org。
```

**Review by EmmyMiao87 — APPROVED** [review]
LGTM

**Pull request merged**